documentation/doxygen/compiler.dxy.in

```diff
@@ -187,7 +208,7 @@ SHORT_NAMES            = NO
 # description.)
 # The default value is: NO.
 
-JAVADOC_AUTOBRIEF      = YES
+JAVADOC_AUTOBRIEF      = NO
```

Comment:
For the time being I would set this back to YES.

This makes Doxygen to correctly recognize brief descriptions when parsing retarget_fs.h API file for File Interface.
Affects API Reference -> File Interface display.

Reply:
Reverted to YES. But we shall think of changing away from javadoc format.